### PR TITLE
HoC 2023 - Update hourofcode.com homepage

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update the hourofcode.com homepage 
- Show multiple calls-to-action in the hero banner based on audience
- Rearrange sections
- Refactored and cleaned up some CSS and views
- Page is fully responsive

**Jira ticket:** [ACQ-989](https://codedotorg.atlassian.net/browse/ACQ-989)

----

## Before
Before

## After
After

--

### The `es` and `la` Spanish versions show this video and banner 

Screenshot 2023-10-24 at 12 18 50 PM

Screenshot 2023-10-24 at 12 18 11 PM

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/styles/040-page.css [hunk 1]
```diff
@@ -535,6 +535,28 @@ button:hover {
   background-color: var(--brand_secondary_dark);
 }
 
+a.link-button.secondary {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/styles/040-page.css:554]: Added these link button classes to be used sitewide

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.scss
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.scss:1]: Refactored this and removed a lot of used styles

Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.scss:1]: Cleaned up and removed a lot of unused styles here

Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.scss:1]: Refactored and removed a lot of unused styles here

**Review by TurnerRiley — APPROVED** [review]

**Pull request merged**